### PR TITLE
{lib}[GCCcore/11.2.0] psycopg2 v2.9.5 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/11d27680c8c7b77e008c92a35db6fb20 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/1afdc6cf38cbf07b8e91e9238bce8196 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
himem06.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/41a9233abffd246e82e6a896c4757169 for a full test report.

File: easybuild/easyconfigs/p/psycopg2/psycopg2-2.9.5-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('PostgreSQL', '13.4'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/psycopg2/psycopg2-2.9.5-GCCcore-11.2.0.eb:29]: should we add a path sanity check, or is that redundant with `sanity_pip_check`?
```suggestion

sanity_check_paths = {
    'files': [],
    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s'],
}

```

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
Going in, thanks @branfosj!

**Pull request merged**